tests/conftest.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+
+
+@pytest.fixture
+def my_tmp_path(tmp_path):
+    return tmp_path
+
+
+@pytest.fixture
+def construct_wrong_fake_configs(my_tmp_path):
+    wrong_fake_configs = {
+        "paths": {
+            "ymldb": my_tmp_path / "data/ymldb",
+            "logs": {"eko": my_tmp_path / "logs/eko", "fk": ["something", "wrong"]},
+        },
+        "root": my_tmp_path,
+    }
+    return wrong_fake_configs
+
+
+@pytest.fixture
+def construct_fake_configs_incomplete(my_tmp_path):
+    fake_configs_incomplete = {
+        "paths": {
+            "ymldb": my_tmp_path / "data/ymldb",
+            "logs": {"eko": my_tmp_path / "logs/eko"},
+        },
+        "root": my_tmp_path,
+    }
+    return fake_configs_incomplete
+
+
+@pytest.fixture
+def construct_fake_configs(my_tmp_path):
```

Comment:
Here and in those above: in a fixture the action is implicit, also because the function is invoked and replaced with the result (e.g. PyTest provides `tmp_path`, not `construct_tmp_path`).
It is different for factory fixtures (that return functions), but these are not factories https://docs.pytest.org/en/7.2.x/how-to/fixtures.html#factories-as-fixtures